config/api.yml

```diff
@@ -1124,10 +1128,6 @@
       :get:
       - :name: show
       - :identifier: miq_cloud_networks_view
-    :load_balancers_subcollection_actions:
-      :get:
-      - :name: show
-        :identifier: load_balancer_show
```

Comment:
ok, so I take it the portion of the PR fixing the bug was adding the subresource_actions get/read role ?